### ISSUE TITLE
Wiggle expression

### ISSUE THREAD

**Issue by MJholland** [airbnb/lottie-web#901]
Hey, not so much an issue but an idea. I didn't realize that the wiggle expression wasn't supported so I'm currently going through and converting all those instances with keyframes (animation>keyframe assistant>convert expression to keyframes). Is there someway Bodymovin could automate this when rendering? It would allow for wiggle to be kind of supported.

**Comment by bodymovin** [comment]
It's difficult to identify when an expression uses wiggle. And converting the expression to keyframes is rather invasive on the user's project.
I'd rather let that decision to the user.
I hope at some point I'll be able to get a wiggle effect close enough to the one in AE.

**Comment by herrmanneffe** [comment]
Hi @bodymovin,
I also found the same problems using the expression Wiggle.

I noticed that the amplitude value seems to work, in fact the animation changes by entering different values.

While the frequency value does not respond at all. It is much faster than it should be.

Reading this article on Expression Speed and Frequency Control I found many similarities in the examples given and in the malfunctions described.

Bodymovin seems to have the same problems in controlling frequency and speed in expressions.

Maybe it can be a useful or inspirational clue to find some bug.

This is the article: http://motionscript.com/articles/speed-control.html#wiggle

And these are the same experiments of the article, but using bodymovin: http://www.federicoherrmann.it/wiggle-test/
